functions/on-new-comment-send-email.js

```diff
@@ -86,19 +86,30 @@ exports.onNewCommentSendEmail = functions.firestore.document('/projects/{project
     if (task.assignee && task.assignee.id && task.assignee.id !== task.creator.id) {
       userPromises.push(getUserInfo(task.assignee.id));
     }
+    task.listeners.forEach((listener)=> userPromises.push(getUserInfo(listener)) );
 
     const usersData = await Promise.all(userPromises);
-    const creator = usersData[0].data();
+   /* const creator = usersData[0].data();
     const languageCreator = creator.language || 'he'; //Defaults to hebrew;
     const mailPromises = [];
 
     // Send the emails
     mailPromises.push(mailgun.messages().send(getEmailParams(creator.email, languageCreator)));
```

Comment:
Better remove comments if we don't use them (For that we have source control) - or otherwise if there is a reason - add why it's being left there